packages/server/src/api/rest/index.ts

```diff
@@ -1598,7 +1606,7 @@ class RequestHandler extends APIHandlerBase {
                         : this.makePrismaIdFilter(info.idFields, value);
                 return { some: filterValue };
             } else {
-                return { is: this.makePrismaIdFilter(info.idFields, value) };
+                return { is: this.makePrismaIdFilter(info.idFields, value, false) };
```

Comment:
Good catch.

I believe Prisma's rule is that the `id1_id2: { ... }` form of filter is only required in a "unique where clause" (used in `findUnique`, `update`, etc).

The filter made by the `makeFilterValue` function are not used in the unique context, so I believe all `makePrismaIdFilter` call inside this function should have `false` as the second parameter.

Reply:
That makes sense. Made this change now.